### DISCUSSION TITLE
PS2 SYSCON Setting

### DISCUSSION THREAD

**Discussion by KatsutoKiyojin** [aldostools/webMAN-MOD#815] [Ideas]
So after updating to evilnat I saw they included an option to set the ps2 fan mode in cobra to default fan mode. This would be a nice feature to keep the thermals low during lvl2 kernel mode but keep syscon fan in charge in case things get dicey in lvl1 mode. I had this a few times on my older launch model. I was thinking of the button being under the already existing ps2 setting not to confuse users too much

**Comment by aldostools** [comment]
Could you elaborate how to implement your idea? Maybe with some C/C++ code or p-code. 

I really didn't understand what you exactly want. The PS2 fan speed is set to a constant value on XMB, so what could be "dicey" in lvl1 mode? If you play PS2 games with SYSCON fan settings, generally you will end with overheating unless you customize the SYSCON table:
https://www.psx-place.com/threads/modifying-syscon-fan-settings-better-than-webman.34626/

**Reply by KatsutoKiyojin** [reply]
I think I misunderstood somewhere in the documentation. I thought normal syscon temps were "safe" for non-faulty units. I assumed I could set it safely for PS2 mode though a fan does suffice. I may look into editing those tables, though as that is more along the lines of what I would want. My B for my confusion
